invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/deposit.less

```diff
@@ -21,6 +21,19 @@
     }
   }
 
+  .ui.form .field .form-group label{
+    font-weight: normal;
+  }
+
+  /**targets the top-level fields, id=top-level-fields is injected in react.*/
+  label#top-level-fields{
+    font-size: 14px;
+  }
+  /**targets the top-level resource_type field,
+  this is because this field (label) does not have the id.*/
+  .required.field label[for="metadata.resource_type"]{
+    font-size: 14px;
+  }
```

Comment:
See https://github.com/inveniosoftware/react-invenio-deposit/pull/197#discussion_r570997932 if we can get a way for us not to repeat these one-offs...  If the strategies there don't work, then let's go with this (but with classes :wink: )

Reply:
yes- you are right, i was thinking the same while working on it. Mybe i can hack around the css and target each and every ```for=""``` attributes of the labels and add the styles - would that make sense?